attestation-agent/attester/src/tdx/mod.rs

```diff
@@ -54,13 +55,17 @@ fn runtime_measurement_extend_available() -> bool {
     true
 }
 
+pub const DEFAULT_EVENTLOG_PATH: &str = "/run/attestation-agent/eventlog";
+
 #[derive(Serialize, Deserialize)]
 struct TdxEvidence {
     // Base64 encoded CC Eventlog ACPI table
     // refer to https://uefi.org/specs/ACPI/6.5/05_ACPI_Software_Programming_Model.html#cc-event-log-acpi-table.
     cc_eventlog: Option<String>,
     // Base64 encoded TD quote.
     quote: String,
+    // Eventlog of Attestation Agent
+    aa_eventlog: Option<String>,
```

Comment:
the open here is do we want to keep `cc_eventlog` and `aa_eventlog` separate or they get sent out in a unified structure? The NELR format used here can work independent of `cc_eventlog` when `INIT`ed properly but it might still be beneficial for the verifier to use one structure only.

FWIW, while working with ITA I discovered they have a rather special format for the `event_log` that needs to be passed to it:

```
[{"rtmr":{"index":1,"bank":"SHA384"},"rtmr_events":[{"type_id":"0x8000000b","type_name":"EV_EFI_HANDOFF_TABLES2","tags":["TdxTable"],"measurement":"0b8772e5b0b41b83e6044a68397e02f49fb47066b4fbe4917ea2c45c64f323fdacbb37948f821ebaf8bc9c938ba8a749"},{"type_id":"0x8000000a","type_name":"EV_EFI_PLATFORM_FIRMWARE_BLOB2","tags":["Fv(XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX)"],"measurement":"344bc51c980ba621aaa00da3ed7436f7d6e549197dfe699515dfa2c6583d95e6412af21c097d473155875ffd561d6790"},
...
```

It's similar to what `eventlog-rs` builds but the TCG log parsing is done by the attester/relying party.

Reply:
Yeah as how this NELR could be embedded inside a CELR is not finally decided, I think it is fair to use a separate `AAEL` to carry such NELR information. Once kernel community reached an agreement, we could merge `AAEL` into `CCEL` in a standard way without any extra assumptions.